### ISSUE TITLE
to_nice_yaml filter fails with indent argument

### ISSUE THREAD

**Issue by timw** [ansible/ansible#17055]
##### ISSUE TYPE

- Bug Report
##### COMPONENT NAME

to_nice_yaml filter
##### ANSIBLE VERSION

```
ansible 2.1.0.0
Python 2.7.11
```
##### SUMMARY

{{ variable | to_nice_yaml(indent=4) }} fails with an error message.

The docs at http://docs.ansible.com/ansible/playbooks_filters.html indicate that the indentation of to_nice_yaml can be controlled by providing an indent named argument to the to_nice_yaml filter, but this doesn't work.
##### STEPS TO REPRODUCE

```

---

- hosts: localhost
  vars:
    example:
      host: smtp.example
      port: 587
      user: example
      pass: examplepass
      domain:
        something: email.example
        other: aimail.other.com

  tasks:

    - name: Format YAML
      template: src=content.j2 dest=./to_nice_yaml.output.yml


Content of content.j2:

example:
   {{ example | to_nice_yaml(indent=4) }}

```

##### EXPECTED RESULTS

Indented YAML output created in output file.
##### ACTUAL RESULTS

Playbook fails with following error:

```
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "invocation": {"module_args": {"dest": "./to_nice_yaml.output.yml", "src": "content.j2"}, "module_name": "template"}, "msg": "AnsibleError: Unexpected templating type error occurred on (example:\n   {{ example | to_nice_yaml(indent=4) }}\n): dump() got multiple values for keyword argument 'indent'"}
```

**Comment by timw** [comment]
The obvious workaround is `{{ example | to_nice_yaml | indent(4, False) }}`.

**Comment by timw** [comment]
Looks like this was actually implemented in devel in #10008, but isn't in a stable release and the docs don't mark the feature as "New in 2.2", so this is a doc-bug.

**Comment by dagwieers** [comment]
@timw And for documentation changes everyone can make improvements too !
Here's a direct link to edit the document and make a pull request:
https://github.com/ansible/ansible/edit/devel/docsite/rst/playbooks_filters.rst

Welcome new Ansible developer ! ;-)

**Comment by resmo** [comment]
Fixed by #17085

**Comment by dagwieers** [comment]
@timw Thanks for your contribution !
